tests/test_cache_account_info.py

```diff
@@ -172,10 +172,12 @@ def test_can_refresh_entire_bucket_name_cache():
     bucket_id_or_none = cacheAccountInfo.get_bucket_id_or_none_from_bucket_name("some-name")
     bucket_id_or_none2 = cacheAccountInfo.get_bucket_id_or_none_from_bucket_name("other-name")
     random_id_or_none = cacheAccountInfo.get_bucket_id_or_none_from_bucket_name("something-random")
+    name_ids = cacheAccountInfo.list_bucket_names_ids()
```

Comment:
sorry one more comment here, I think probably this warrants its own test similar to `test_get_bucket_id_when_bucket_name_set` and `test_get_bucket_id_when_bucket_name_not_set`

Exceeppttt, reading this test, I can't for the life of me figure out why the list is `[("some-name", "new-id")]` and not `[("some-name", "new-id"), ("something-random", "some-random-id")]` 

Can you explain why? 🤔

Reply:
ahhhhhhh there's a bug in this method, the cache for all names isn't updated 😅

Reply:
Fixed in #110 -- thanks for uncovering this, unintentionally 😛

Reply:
I re-added the modification. Seemed easier than going back and forth.